src/__tests__/WorldPage.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import {
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import WorldPage from '../Pages/WorldPage';
+
+import store from '../redux/store';
+
+afterEach(cleanup);
+
+it('renders World page with redux', () => {
+  const { asFragment } = render(
+    <Provider store={store}>
+      <Router>
+        <WorldPage />
+      </Router>
+    </Provider>,
+  );
```

Comment:
- [ ] There are a couple of failing tests. Kindly check and fix them all.

Reply:
Hello Mike, 

Thanks a lot for finding this issue, it was caused because the provider of the store moved to another file different than APP.js.